debian/control.in

```diff
@@ -6,26 +6,26 @@ Build-Depends: debhelper (>= 6),
     autoconf (>= 2.63), automake, libboost-python-dev, libgl1-mesa-dev,
     libglu1-mesa-dev, libgtk2.0-dev, libmodbus-dev (>= 3.0), libudev-dev,
     libncurses-dev, libreadline-dev, libusb-1.0-0-dev, libxmu-dev,
-    libxmu-headers, python (>= 2.6.6-3~), python-dev (>= 2.6.6-3~),
+    libxmu-headers, python (>= 2.6.6), python-dev (>= 2.6.6),
     cython (>= 0.19), dh-python,
     pkg-config, psmisc, python-tk, libxaw7-dev, libboost-serialization-dev,
-    libzmq3-dev (>= 4.0.4+dfsg-1~jessie), libczmq-dev (>= 4.0.2-10~jessie), libjansson-dev (>= 2.5),
+    libzmq3-dev (>= 4.0.4), libczmq-dev (>= 4.0.2), libjansson-dev (>= 2.5),
```

Comment:
The czmq / zeromq libraries in the machinekit repo for Jessie, are actually versioned with the prefix `~jessie`.
As are all our other distro specific libs  http://deb.machinekit.io/debian/pool/main/c/czmq/

I found that when installing the packages, they failed if you just used the version number on its own for the dependency.  Hence the separate control.in files for Stretch.

I could amend the changelog and build vanilla versions, albeit it would be unclear which distro they were built for, which I think is the origin of the practice.

Reply:
Ok, I'm testing.  If removing these breaks apt dependency resolution somehow, the problem is likely in the dependencies, not here.

I agree the `~jessie` suffix should be maintained.

Reply:
Because the specific `machinekit-dev` packages have been done away with, I am wondering if we should just make a package dependency of `libczmq-dev` (which has this same name in Jessie and Stretch, with no version) so that it is ready to build components that include any machinetalk related code.

It will add about 20 headers, 2 static build libs and a couple of symlinks to the package.
However specifying it pulls in libczmq4 and libzmq3-dev (which pulls libzmq3).  
python-zmq will pull in libzmq5 because that is what it uses, but it does not clash.

We don't need to specify a version and Jessie will just find the later version in the machinekit repo and pull the right files.

I had not realised that the Stretch rt-preempt kernels had been backported for Jessie, that makes that issue much simpler now as you have done it.
https://packages.debian.org/jessie-backports/linux-image-rt-amd64
https://packages.debian.org/jessie-backports/linux-image-rt-686-pae

Reply:
[I had to ask for help with the `machinekit-dev` history off-thread from @ArcEye.]

About the kernels, if those could be pulled into the Jessie repo, the `debian/control.rt-preempt*.in` files could be merged back together, since the kernel package naming scheme would be consistent again.

Reply:
I would prefer to just amend the documentation for installing a kernel for Jessie i386 and amd64.

If the user adds jessie-backports they will get the latest debian supported rt-preempt kernel and it will be something else we don't have to do 😄 

So go ahead and merge the control files, I will make a docs change, because it will work before this is merged.

Regards the `-dev` stuff, in days when it is becoming hard to buy a SD card smaller than 8GB
(and a 4GB one costs not much less) space is not the issue.

Why people insist on using boards with 512MB RAM and puny processors, instead of proper computers IS an issue to my mind, but that is a completely different matter 😜

Reply:
The PR for the docs is done and at https://github.com/machinekit/machinekit-docs/pull/268

Reply:
...and rt-preempt control files are now merged with the common Debian-format rt-preempt kernel package naming.